samza-core/src/main/java/org/apache/samza/config/StorageConfig.java

```diff
@@ -74,9 +74,12 @@
   static final String INMEMORY_KV_STORAGE_ENGINE_FACTORY =
       "org.apache.samza.storage.kv.inmemory.InMemoryKeyValueStorageEngineFactory";
 
-  // Internal config to clean storeDirs of a store on container start. This is used to benchmark bootstrap performance.
+  // Internal config to clean storeDirs of a logged store on container start. This is used to benchmark bootstrap performance.
   static final String CLEAN_LOGGED_STOREDIRS_ON_START = STORE_PREFIX + "%s.clean.on.container.start";
 
+  // Internal config to clean storeDirs of a logged store on container start. This is used to benchmark bootstrap performance.
+  static final String RETAIN_NONLOGGED_STOREDIRS_ON_START = STORE_PREFIX + "%s.retain.on.container.start";
```

Comment:
should we name this config to reflect the non-logged only case ? `%s.retain.nonlogged.on.container.start`